src/DatePicker/DatePickerDialog.js

```diff
@@ -136,7 +138,7 @@ class DatePickerDialog extends Component {
           repositionOnUpdate={true}
           open={open}
           onRequestClose={this.handleRequestClose}
-          style={styles.dialogBodyContent}
+          style={Object.assign(styles.dialogBodyContent, containerStyle)}
```

Comment:
Oh wait!
What is this `<div {...other} ref="root">` for? The `{...other}` property is spreaded twice 😨 .
Could we just kill the root `div` and use `style` instead of `containerStyle`?